tests/pipeline_test.py

```diff
@@ -1,4 +1,4 @@
-__author__='thiagocastroferreira'
+__author__ = "thiagocastroferreira"
```

Comment:
Can all assets: metrics, benchmarks, models have a functional test similar to this?

The code will be similar to your unit tests, though it will test the e2e functionality of the module. This can be made a part of a later PR.